source/features/avoid-accidental-submissions.tsx

```diff
@@ -14,27 +15,39 @@ function onKeyDown(event: delegate.Event<KeyboardEvent, HTMLInputElement>): void
 		|| event.ctrlKey
 		|| event.metaKey
 		|| event.isComposing // #4323
-		|| select.exists('.suggester', event.delegateTarget.form!) // GitHub’s autocomplete dropdown
+		|| select.exists([
+			'.suggester', // GitHub’s autocomplete dropdown
+			'.rgh-avoid-accidental-submissions.my-1',
+		],
+		event.delegateTarget.form!)
 	) {
 		return;
 	}
 
+	const message = <p className="rgh-avoid-accidental-submissions my-1">A submission via <kbd>enter</kbd> has been prevented. You press <kbd>enter</kbd> again or use <kbd>ctrl</kbd>-<kbd>enter</kbd> next time</p>;
+
+	const focusedInput = select(inputElements, event.delegateTarget.form!)!;
+
+	if (select.exists('.btn-primary[type="submit"]:not([disabled])')) {
```

Comment:
This is only preventing the message, but it’s still stopping the submission. The message should always be shown if the submission has been prevented 
```suggestion
	if (!select.exists('.btn-primary[type="submit"]:not([disabled])')) {
	   return
	}
```

Also the selector just looks for that button anywhere on the page, but it should be limited to the form instead.

Reply:
Thank you. So I changed it into:

```js
	if (!(select.exists('.btn-primary[type="submit"]:not([disabled])'), event.delegateTarget.form!)) {
		return;
	}
```